modules/auto-update/auto-updater-module-interface.js

```diff
@@ -0,0 +1,24 @@
+const OTAutoUpdater = require('./implementation/ot-auto-updater');
+
+module.exports = class AutoUpdater {
```

Comment:
Update the name of the class to AutoUpdaterModuleInterface
Also move module.exports in the end of the file:
class AutoUpdaterModuleInterface {
}
module.exports = AutoUpdaterModuleInterface